cloudify-web-monitoring-plugin/blueprints/AKS/cluster.yaml

```diff
@@ -0,0 +1,386 @@
+tosca_definitions_version: cloudify_dsl_1_3
```

Comment:
No I don't think so! Can we use the existing Kubernetes examples that we already have to support? I don't want to support more Kubernetes examples. Support yet another? No freaking way!

Reply:
Changed to use AKS example as component